README.md

```diff
@@ -1,32 +1,46 @@
 # LCache
 Foundation Library for Coherent, Multi-Layer Caching
 
-## Testing
-
 [![Build Status](https://travis-ci.org/lcache/lcache.svg?branch=master)](https://travis-ci.org/lcache/lcache)
 [![Coverage Status](https://coveralls.io/repos/github/lcache/lcache/badge.svg?branch=master)](https://coveralls.io/github/lcache/lcache?branch=master)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/lcache/lcache/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/lcache/lcache/?branch=master)
+[![Documentation](https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat)](https://lcache.github.io/lcache/)
+[![API](https://img.shields.io/badge/api-latest-brightgreen.svg?style=flat)](https://lcache.github.io/lcache/api/master)
+[![License](https://poser.pugx.org/lcache/lcache/license)](https://packagist.org/packages/lcache/lcache)
 
-### On Fedora
+## Testing
 
  1. Install packages:
+ 
+    ** On Fedora: **
 
     ```
     sudo dnf install -y php-cli composer php-phpunit-PHPUnit php-phpunit-DbUnit php-pecl-apcu
     ```
+    
+    ** On MacOS: **
+    
+    ```
+    brew install php70-xdebug 
+    brew install php70-opcache
+    brew install php70-apcu
+    ```
 
  2. Enable APCu caching for the CLI:
 
     ```
     echo "apc.enable_cli=1" | sudo tee -a /etc/php.d/40-apcu.ini
```

Comment:
Yes, the setting seems to be named the same thing on macOS.  However, it is enabled by default in `/usr/local/etc/php//7.0/conf.d/ext-apcu.ini`, so this step is not always necessary.